### PR TITLE
LPS-132726 Add scroll inside of the metrics modal

### PR DESCRIPTION
**Pull request by MariMalta**

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
Hey @MariMalta and @seltong it would be nice if you add an integration test to cover this scenario. Thank you.

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @MariMalta and @seltong it would be nice if you add an integration test to cover this scenario. Thank you.

**Comment by seltong** [comment]
Just started reviewing :)

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java [hunk 1]
```diff
@@ -179,6 +179,8 @@ public Instance getProcessInstance(Long processId, Long instanceId)
 		TopHitsAggregation topHitsAggregation = _aggregations.topHits(
 			"topHits");
 
+		topHitsAggregation.setSize(100);
```
[the remark this answers is not in the document]
Comment (MariMalta) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java:182]: Got it! I'm going to change this now! Thank you!

**Comment by seltong** [comment]
Just started reviewing :)

**Review by naraandrade — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java [hunk 2]
```diff
@@ -85,6 +85,28 @@ public void tearDown() throws Exception {
...
+	@Test
+	public void testGetProcessInstance() throws Exception {
+		Instance instance = randomInstance();
+
+		instance.setSlaResults(
+			new SLAResult[] {
+				_toSLAResult(true, SLAResult.Status.STOPPED),
+				_toSLAResult(true, SLAResult.Status.PAUSED),
+				_toSLAResult(true, SLAResult.Status.RUNNING),
+				_toSLAResult(true, SLAResult.Status.NEW)
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java:98]: Hi @seltong, please, order the SLAResults by its status in alphabetical order.

Thank you in advance!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by seltong** [comment]
Just started reviewing :)

**Comment by seltong** [comment]
Just started reviewing :)

**Comment by rafaprax** [comment]
ci:forward

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MariMalta** [comment]
ci:forward

**Comment by MariMalta** [comment]
ci:forward

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Pull request closed without merging**